inc/session_handler/namespace.php

```diff
@@ -0,0 +1,8 @@
+<?php
+/**
+ * Altis Cloud Session handler.
+ *
+ * @package altis/cloud
+ */
+
+require_once __DIR__ . '/class-disallowed-session-handler.php';
```

Comment:
I think this maybe counts as a side-effect 🤔  Either way this require can be moved to the module's `autoload.classmap` property in the `composer.json`.

Follow the pattern set out here:

https://github.com/humanmade/altis-cloud/blob/master/composer.json#L12-L27